src/components/paradox/results/personality/career/inline-jobs/job.js

```diff
@@ -0,0 +1,67 @@
+import PropTypes from "prop-types";
+import {faLocationDot} from "@fortawesome/free-solid-svg-icons";
+import Icon from "lib/helpers/icon";
+import DangerousHTML from "lib/helpers/dangerous-html";
+import style from "./style.scss";
+
+function InlineJob({job, jobSource, jobSourceURL, translate}) {
+  if(!job || Object.keys(job).length === 0) {
+    return (
+      <div className={`${style.inlineJob} ${style.empty}`}>
+        <div>
+          <DangerousHTML html={translate("no_jobs", {job_source: jobSource})} />
+        </div>
+        <div>
+          <a
+            className={style.discoverJobsButton}
+            href={jobSourceURL[jobSource]}
```

Comment:
Is this only used here? We could just define it for this component instead of passing it down in all the others

Reply:
Im using jobSourceURL to validate jobSource in the parent component. If a value not in jobSourceURL is passed in then the parent component returns null.